tfexec/options.go

```diff
@@ -172,6 +172,14 @@ func PluginDir(pluginDir string) *PluginDirOption {
 	return &PluginDirOption{pluginDir}
 }
 
+type ReattachOption struct {
+	info string
+}
+
+func Reattach(reattach string) *ReattachOption {
```

Comment:
this should probably take the structure, not a string, similar to all our other terraform-json handling